index.js

```diff
@@ -141,44 +173,64 @@ const line = []
 
 
 // *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 1", "color: red")
 
-// console.log(takeATicketNumber(line))
-// // => `Welcome. You are ticket number 1`
+  var ticketNumber = 1
+  function takeATicketNumber(line) {
+    line.push(ticketNumber)
+    var newMsg = "Welcome. You are ticket number " + ticketNumber
+      ticketNumber++;
+      return newMsg;
+  }
```

Comment:
Good! One suggestion is to stick with `let` and `const` for variable declarations. Since ES6, this has been the best practice because it helps prevent some scoping issues related with using `var`. Check out[ this video ](https://www.youtube.com/watch?v=wtbZCWnZIyQ&feature=emb_title)to learn more about why